AcademyChallenge/Data/API/EmojiNetwork.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  Network.swift
+//  AcademyChallenge
+//
+//  Created by Keven Esmael on 30/09/2022.
+//
+
+import Foundation
+
+
+class EmojiNetwork {
+	
+	
+	static func initialize() {
+	 URLSession.shared.configuration.urlCache?.diskCapacity = 100 * 1024 * 1024
+	 print("Current disk cache capacity: \(URLSession.shared.configuration.urlCache?.diskCapacity)")
+ }
+	
+	func executeNetworkCall<ResultType: Decodable>(_ call: APIProtocol, _ resultHandler: @escaping (Result<ResultType, Error>) -> Void) {
```

Comment:
It can be moved to a more generic object (check the comment on the `AvatarNetwork` file

Reply:
solved